types/mapcmd/map.go

```diff
@@ -48,6 +75,35 @@ func getMap(ctx context.Context, clientConfig *hazelcast.Config, mapName string)
 	return
 }
 
+func NewUse() *cobra.Command {
+	//var mapName, mapKey string
+	cmd := &cobra.Command{
+		Use:   "use map-name",
+		Short: "sets default map name",
+		Example: "map use m1		# sets the default map name to m1 unless set explicitly\n" +
+			"map get --key k1	# \"--name m1\" is inferred\n" +
```

Comment:
No need to escape quotes if you use backtick strings. There are a few more below.

Reply:
I really don't like the formatting of multiline strings. But let's go with your way for to match the code base.